src/js/webrequest.js

```diff
@@ -1125,6 +1126,18 @@ function dispatcher(request, sender, sendResponse) {
     break;
   }
 
+  case "showTrackingDomainsSection": {
+    badger.getSettings().setItem("showExpandedTrackingSection", true);
+    sendResponse();
+    break;
+  }
+
+  case "hideTrackingDomainsSection": {
+    badger.getSettings().setItem("showExpandedTrackingSection", false);
+    sendResponse();
+    break;
+  }
+
```

Comment:
We could just reuse `"updateSettings"` instead.

Reply:
I can easily switch to that. Is it worth breaking away from how the other similar methods in this chunk handle updating a setting though?

Reply:
>Is it worth breaking away from how the other similar methods in this chunk handle updating a setting though?

Do you mean that we don't yet use the `updateSettings` message in the popup? Yeah, we don't need to add more one-off messages to toggle a setting going forward.

Reply:
Ah gotcha, fixed this